fmriprep/data/boilerplate.bib

```diff
@@ -0,0 +1,260 @@
+@article{fmriprep1,
+	author = {Esteban, Oscar and Markiewicz, Christopher and Blair, Ross W and Moodie, Craig and Isik, Ayse Ilkay and Erramuzpe Aliaga, Asier and Kent, James and Goncalves, Mathias and DuPre, Elizabeth and Snyder, Madeleine and Oya, Hiroyuki and Ghosh, Satrajit and Wright, Jessey and Durnez, Joke and Poldrack, Russell and Gorgolewski, Krzysztof Jacek},
+	title = {FMRIPrep: a robust preprocessing pipeline for functional MRI},
+	year = {2018},
+	doi = {10.1101/306951},
+	journal = {bioRxiv}
+}
+
+@article{fmriprep2,
+ 	author = {Esteban, Oscar and Blair, Ross and Markiewicz, Christopher J. and Berleant, Shoshana L. and Moodie, Craig and Ma, Feilong and Isik, Ayse Ilkay and Erramuzpe, Asier and Kent, James D. andGoncalves, Mathias and DuPre, Elizabeth and Sitek, Kevin R. and Gomez, Daniel E. P. and Lurie, Daniel J. and Ye, Zhifang and Poldrack, Russell A. and Gorgolewski, Krzysztof J.},
+ 	title = {FMRIPrep 1.1.2},
```

Comment:
We need to figure out how to cite the right version here. Especially considering that the zenodo handle is created after the release is made.

Reply:
I think I may have the solution. I just found this on the Zenodo record of fmriprep:

> **Cite all versions?** You can cite all versions by using the DOI 10.5281/zenodo.852659. This DOI represents all versions, and will always resolve to the latest one. Read more.

We could reset the version in the bib file during deployment. Would that reply this concern?